package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "understrap",
-  "version": "0.8.3",
+  "version": "0.8.9",
   "description": "WordPress Theme framework",
   "main": "index.js",
   "scripts": {
-    "postinstall": "gulp copy-assets"
-  },
-  "engines": {
-    "npm": ">=2.1.8"
+    "postinstall": "gulp copAssets",
```

Comment:
There is a typo here, it is `copyAssets`

Reply:
After correcting the postinstall command to gulp copyAssets, no other error?

Reply:
seems that now is fixed